packages/astro/src/default/components/MobileContentToggle.astro

```diff
@@ -0,0 +1,50 @@
+---
+
+---
+
+<div class="h-12 sm:hidden"></div>
+<view-toggle
+  class="fixed sm:hidden h-12 bottom-0 w-full bg-tk-elements-app-backgroundColor border-t border-tk-elements-app-borderColor flex justify-center items-center z-60 text-tk-elements-app-textColor"
+>
+  <button aria-hidden="true">Tutorial</button>
+  <button aria-pressed="true" class="rounded-full w-8 h-4 p-0.5 bg-gray-4 dark:bg-gray-6 mx-2 relative">
+    <span class="inline-block transition-all absolute top-0.5 left-0.5 rounded-full bg-white dark:bg-gray-2 w-3 h-3"
+    ></span>
+  </button>
+  <button aria-hidden="true">Editor</button>
+</view-toggle>
+<script>
+  import { viewStore, type View } from '../stores/view-store';
+
+  class ViewToggle extends HTMLElement {
+    constructor() {
+      super();
+
+      const [tutorialBtn, toggleButton, editorButton] = this.querySelectorAll(
+        ':scope > button',
+      ) as NodeListOf<HTMLButtonElement>;
+
+      const toggle = toggleButton.querySelector('span')!;
+
+      tutorialBtn.onclick = () => setView('content');
+      editorButton.onclick = () => setView('editor');
+      toggleButton.onclick = () => setView(viewStore.get() === 'content' ? 'editor' : 'content');
```

Comment:
Is a touch identical to a click?

Reply:
Yes! It's actually even more than [a mouse click](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event):

> An element receives a click event when any of the following occurs:
>
> * a pointing-device button (such as a mouse's primary button) is both pressed and released while the pointer is located inside the element.
> * a touch gesture is performed on the element
> * the Space key or Enter key is pressed while the element is focused